modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/EditObjectFolder.tsx

```diff
@@ -101,27 +101,57 @@ export default function EditObjectFolder({
 		});
 	};
 
-	const updateModelBuilderStructure = async (
-		newObjectRelationshipId: number
+	const onAfterAddObjectRelationship = async (
+		newObjectRelationship: ObjectRelationship
 	) => {
 		const payload = await getUpdatedModelBuilderStructurePayload(
 			baseResourceURL,
 			selectedObjectFolder.name
 		);
 
+		if (
+			newObjectRelationship.objectDefinitionId1 !==
+			newObjectRelationship.objectDefinitionId2
```

Comment:
When the choice regarding the use of Object Definition ID or Object Definitions External Reference Code does not harm the code, I believe we should choose to use the external reference code.